portfolio/src/main/webapp/script.js

```diff
@@ -41,3 +41,36 @@ function toggleDescription(sectionId) {
   }
   toggleClass(sectionElement, "collapsed");
 }
+
+/**
+ * Change the slide to the current one + n
+*/
+function changeSlide(n, slider) {
+  if (!n || !Number.isInteger(n)) {
+    throw "changeSlide first argument is not an integer.";
+  }
+  let totalSlides = 0;
+  let currentlyDisplayed;
+  for (const elem of slider.children) {
+    const classes = elem.className.split(" ");
+    if (classes.indexOf("slide") !== -1) {
+      if (classes.indexOf("hidden") === -1) {
+        currentlyDisplayed = totalSlides;
+        elem.className += " hidden";
+      }
+      totalSlides++;
+    }
+  }
```

Comment:
I thought about making this part a separate function, but then I should have returned two values for example using an array and I decided to keep it so it looks cleaner.
What do you think?

Reply:
as mentioned in the the standup, you can e.g. return an object.